load_easybuild_module.sh

```diff
@@ -0,0 +1,108 @@
+# Script to load the environment module for a specific version of EasyBuild.
+# If that module is not available yet, the current latest EasyBuild version of EasyBuild will be installed,
+# and used to install the specific EasyBuild version being specified.
+#
+# This script must be sourced, since it makes changes in the current environment, like loading an EasyBuild module.
+
+set -o pipefail
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 <EasyBuild version>" >&2
+    exit 1
+fi
+
+# don't use $EB_VERSION, since that enables always running 'eb --version'
+EB_VERSION=${1}
+
+# make sure that environment variables that we expect to be set are indeed set
+if [ -z "${TMPDIR}" ]; then 
+    echo "\$TMPDIR is not set" >&2
+    exit 2
+fi
+
+if [ -z "${EB}" ]; then 
+    echo "\$EB is not set" >&2
+    exit 2
+fi
+
+# make sure that utility functions are defined (cfr. scripts/utils.sh script in EESSI/software-layer repo)
+type check_exit_code
+if [ $? -ne 0 ]; then
+    echo "check_exit_code function is not defined" >&2
+    exit 3
+fi
+
+echo ">> Checking for EasyBuild module..."
+
+ml_av_easybuild_out=${TMPDIR}/ml_av_easybuild.out
+module avail 2>&1 | grep -i easybuild/${EB_VERSION} &> ${ml_av_easybuild_out}
+
+if [[ $? -eq 0 ]]; then
+    echo_green ">> Module for EasyBuild v${EB_VERSION} found!"
+else
+    echo_yellow ">> No module yet for EasyBuild v${EB_VERSION}, installing it..."
+
+    EB_TMPDIR=${TMPDIR}/ebtmp
+    echo ">> Temporary installation (in ${EB_TMPDIR})..."
+    pip_install_out=${TMPDIR}/pip_install.out
+    pip3 install --prefix ${EB_TMPDIR} easybuild &> ${pip_install_out}
```

Comment:
Does this (usually) install the latest release?

Reply:
Yes, if no version is specified (like `pip install ... easybuild==4.7.2`), then the latest release available in PyPI is automatically installed